packages/static-wado-webserver/lib/routes/study.mjs

```diff
@@ -0,0 +1,11 @@
+import serverSCPFallbackController from "../controllers/serverSCPController.mjs";
+
+/**
+ * Set studies (/studies) routes.
+ *
+ * @param {*} router root entry point for studies routes.
+ */
+export default function setRoutes(router) {
+  // fallback route to external SCP
+  router.get("/studies/:studyUID/series/*.json.gz", serverSCPFallbackController({ studyInstanceUIDPattern: "studyUID" }, true));
```

Comment:
This route should only be added if it is configured in the current configuration, and needs to get the configuration object included because it should be getting the remote SCP as well as local SCU configuration from the configuration object.